client/src/components/BookCardShort/BookCardShort.js

```diff
@@ -121,10 +125,41 @@ const DeleteButton = styled.div`
   right: 7px;
   bottom: 2px;
 `
+
+const ReadMoreButton = styled.div`
```

Comment:
Why did you use a `div` with `role="button"` here? You could style it exactly the same way using a real `<button>` element, since a div should only be used, if no other element fits.

Reply:
True! But as this button is placed within a form, it would always submit my form when clicked. So thats why it has to be a div

Reply:
Ah, alright then! :)